packages/rn-tester/js/RNTesterAppShared.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import {
+  BackHandler,
+  StyleSheet,
+  useColorScheme,
+  View,
+  LogBox,
+} from 'react-native';
+import * as React from 'react';
+
+import RNTesterExampleContainer from './components/RNTesterExampleContainer';
+import RNTesterExampleList from './components/RNTesterExampleList';
+import RNTesterNavBar from './components/RNTesterNavbar';
+import RNTesterList from './utils/RNTesterList';
+import {
+  Screens,
+  initialState,
+  getExamplesListWithBookmarksAndRecentlyUsed,
+  getInitialStateFromAsyncStorage,
+} from './utils';
+import {useAsyncStorageReducer} from './utils/useAsyncStorageReducer';
+import {RNTesterReducer, RNTesterActionsType} from './utils/RNTesterReducer';
+import {RNTesterThemeContext, themes} from './components/RNTesterTheme';
+import {Header} from './components/RNTesterHeader';
+import {RNTesterEmptyBookmarksState} from './components/RNTesterEmptyBookmarksState';
+
+import type {RNTesterTheme} from './components/RNTesterTheme';
+import type {ExamplesList} from './types/RNTesterTypes';
+
+const APP_STATE_KEY = 'RNTesterAppState.v3';
+
+// RNTester App currently uses Async Storage from react-native for storing navigation state
+// and bookmark items.
+// TODO: Add Native Async Storage Module in RNTester
+LogBox.ignoreLogs([new RegExp('has been extracted from react-native')]);
```

Comment:
Can we either add AsyncStorage to this or list all of the libs that throw this individually? Also there's no need for the RegExp constructor, we could just do `/has been extracted from react-native/`.

Reply:
Done!